tests/test_crs/valid_iperf3.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  workload:
+    name: iperf3
+    args:
+      pairs: 1
+      port: 5201
+      transmit_type: time
+      transmit_value: 60
+      omit_start: 0
+      length_buffer: 128K
+      window_size: 64k
+      ip_tos: 0
+      mss: 900
+      streams: 1
+      extra_options_server: ' '
+      extra_options_client: ' '
```

Comment:
Should you test w/ a client option like -u?

Reply:
ack that's right let me add a couple of options to test.